src/main/java/lv/ctco/javaschool/goal/control/TypeConverter.java

```diff
@@ -0,0 +1,41 @@
+package lv.ctco.javaschool.goal.control;
+
+import lv.ctco.javaschool.goal.entity.domain.Goal;
+import lv.ctco.javaschool.goal.entity.dto.GoalDto;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+
+public class TypeConverter {
```

Comment:
In the future, I would advise to split this class into `DtoConverter` and `DateTimeConverter`.